2020-10-05 21:21:50.777494: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_100.dll
Running tests under Python 3.6.12: C:\ProgramData\Anaconda3\envs\keras\python.exe
[ RUN      ] AbcTest.testAbstract
[       OK ] AbcTest.testAbstract
[ RUN      ] AbcTest.testConcrete
[       OK ] AbcTest.testConcrete
[ RUN      ] AbcTest.test_session
[  SKIPPED ] AbcTest.test_session
[ RUN      ] FlattenTest.test_attribute_traversal_key
WARNING:tensorflow:From C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\tensorflow_core\python\ops\resource_variable_ops.py:1630: calling BaseResourceVariable.__init__ (from tensorflow.python.ops.resource_variable_ops) with constraint is deprecated and will be removed in a future version.
Instructions for updating:
If using Keras pass *_constraint arguments to layers.
W1005 21:21:54.497744  8384 deprecation.py:506] From C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\tensorflow_core\python\ops\resource_variable_ops.py:1630: calling BaseResourceVariable.__init__ (from tensorflow.python.ops.resource_variable_ops) with constraint is deprecated and will be removed in a future version.
Instructions for updating:
If using Keras pass *_constraint arguments to layers.
2020-10-05 21:21:54.517118: W tensorflow/python/util/util.cc:299] Sets are not currently considered sequences, but this may change in the future, so consider avoiding using them.
[       OK ] FlattenTest.test_attribute_traversal_key
[ RUN      ] FlattenTest.test_attributes_to_ignore
[       OK ] FlattenTest.test_attributes_to_ignore
[ RUN      ] FlattenTest.test_flatten0 (<function FlattenTest.<lambda> at 0x0000015DB27F27B8>)
[       OK ] FlattenTest.test_flatten0 (<function FlattenTest.<lambda> at 0x0000015DB27F27B8>)
[ RUN      ] FlattenTest.test_flatten1 (<class 'list'>)
[       OK ] FlattenTest.test_flatten1 (<class 'list'>)
[ RUN      ] FlattenTest.test_flatten2 (<class 'tuple'>)
[       OK ] FlattenTest.test_flatten2 (<class 'tuple'>)
[ RUN      ] FlattenTest.test_flatten3 (<function <lambda> at 0x0000015DB27E9F28>)
[       OK ] FlattenTest.test_flatten3 (<function <lambda> at 0x0000015DB27E9F28>)
[ RUN      ] FlattenTest.test_model_discover_submodule
[       OK ] FlattenTest.test_model_discover_submodule
[ RUN      ] FlattenTest.test_module_discover_layer_variable
[       OK ] FlattenTest.test_module_discover_layer_variable
[ RUN      ] FlattenTest.test_session
[  SKIPPED ] FlattenTest.test_session
[ RUN      ] FlattenTest.test_with_path
[       OK ] FlattenTest.test_with_path
[ RUN      ] ForwardMethodsTest.testEntersNameScope_call
2020-10-05 21:21:54.596224: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library nvcuda.dll
2020-10-05 21:21:54.709345: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1618] Found device 0 with properties: 
name: GeForce RTX 2080 Ti major: 7 minor: 5 memoryClockRate(GHz): 1.65
pciBusID: 0000:73:00.0
2020-10-05 21:21:54.710052: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_100.dll
2020-10-05 21:21:54.714043: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cublas64_100.dll
2020-10-05 21:21:54.717456: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cufft64_100.dll
2020-10-05 21:21:54.719078: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library curand64_100.dll
2020-10-05 21:21:54.723923: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusolver64_100.dll
2020-10-05 21:21:54.727175: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusparse64_100.dll
2020-10-05 21:21:54.736253: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudnn64_7.dll
2020-10-05 21:21:54.737233: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1746] Adding visible gpu devices: 0
2020-10-05 21:21:54.737807: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2
2020-10-05 21:21:54.751275: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1618] Found device 0 with properties: 
name: GeForce RTX 2080 Ti major: 7 minor: 5 memoryClockRate(GHz): 1.65
pciBusID: 0000:73:00.0
2020-10-05 21:21:54.751855: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_100.dll
2020-10-05 21:21:54.752201: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cublas64_100.dll
2020-10-05 21:21:54.752540: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cufft64_100.dll
2020-10-05 21:21:54.752879: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library curand64_100.dll
2020-10-05 21:21:54.753226: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusolver64_100.dll
2020-10-05 21:21:54.753570: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusparse64_100.dll
2020-10-05 21:21:54.753913: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudnn64_7.dll
2020-10-05 21:21:54.754713: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1746] Adding visible gpu devices: 0
2020-10-05 21:21:55.696294: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1159] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-10-05 21:21:55.696712: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1165]      0 
2020-10-05 21:21:55.696947: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1178] 0:   N 
2020-10-05 21:21:55.697696: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1304] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 8686 MB memory) -> physical GPU (device: 0, name: GeForce RTX 2080 Ti, pci bus id: 0000:73:00.0, compute capability: 7.5)
WARNING:tensorflow:From C:\ProgramData\Anaconda3\envs\keras\lib\contextlib.py:60: TensorFlowTestCase.test_session (from tensorflow.python.framework.test_util) is deprecated and will be removed in a future version.
Instructions for updating:
Use `self.session()` or `self.cached_session()` instead.
W1005 21:21:55.716256  8384 deprecation.py:323] From C:\ProgramData\Anaconda3\envs\keras\lib\contextlib.py:60: TensorFlowTestCase.test_session (from tensorflow.python.framework.test_util) is deprecated and will be removed in a future version.
Instructions for updating:
Use `self.session()` or `self.cached_session()` instead.
2020-10-05 21:21:55.727143: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1618] Found device 0 with properties: 
name: GeForce RTX 2080 Ti major: 7 minor: 5 memoryClockRate(GHz): 1.65
pciBusID: 0000:73:00.0
2020-10-05 21:21:55.727665: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_100.dll
2020-10-05 21:21:55.728021: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cublas64_100.dll
2020-10-05 21:21:55.728364: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cufft64_100.dll
2020-10-05 21:21:55.728701: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library curand64_100.dll
2020-10-05 21:21:55.729044: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusolver64_100.dll
2020-10-05 21:21:55.729387: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusparse64_100.dll
2020-10-05 21:21:55.729731: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudnn64_7.dll
2020-10-05 21:21:55.730284: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1746] Adding visible gpu devices: 0
2020-10-05 21:21:55.730655: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1159] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-10-05 21:21:55.731008: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1165]      0 
2020-10-05 21:21:55.731228: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1178] 0:   N 
2020-10-05 21:21:55.731727: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1304] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 8686 MB memory) -> physical GPU (device: 0, name: GeForce RTX 2080 Ti, pci bus id: 0000:73:00.0, compute capability: 7.5)
[       OK ] ForwardMethodsTest.testEntersNameScope_call
[ RUN      ] ForwardMethodsTest.testEntersNameScope_concreteFunction
WARNING:tensorflow:Entity <function Function._initialize_uninitialized_variables.<locals>.initialize_variables at 0x0000015DB280FEA0> could not be transformed and will be executed as-is. Please report this to the AutoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: 'NoneType' object has no attribute '_fields'
W1005 21:21:57.469164  8384 ag_logging.py:146] Entity <function Function._initialize_uninitialized_variables.<locals>.initialize_variables at 0x0000015DB280FEA0> could not be transformed and will be executed as-is. Please report this to the AutoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: 'NoneType' object has no attribute '_fields'
2020-10-05 21:21:57.474998: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1618] Found device 0 with properties: 
name: GeForce RTX 2080 Ti major: 7 minor: 5 memoryClockRate(GHz): 1.65
pciBusID: 0000:73:00.0
2020-10-05 21:21:57.475522: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_100.dll
2020-10-05 21:21:57.475869: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cublas64_100.dll
2020-10-05 21:21:57.476214: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cufft64_100.dll
2020-10-05 21:21:57.476551: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library curand64_100.dll
2020-10-05 21:21:57.476893: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusolver64_100.dll
2020-10-05 21:21:57.477241: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusparse64_100.dll
2020-10-05 21:21:57.477588: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudnn64_7.dll
2020-10-05 21:21:57.478149: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1746] Adding visible gpu devices: 0
2020-10-05 21:21:57.478461: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1159] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-10-05 21:21:57.478810: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1165]      0 
2020-10-05 21:21:57.479027: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1178] 0:   N 
2020-10-05 21:21:57.479548: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1304] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 8686 MB memory) -> physical GPU (device: 0, name: GeForce RTX 2080 Ti, pci bus id: 0000:73:00.0, compute capability: 7.5)
WARNING:tensorflow:Entity <function Function._initialize_uninitialized_variables.<locals>.initialize_variables at 0x0000015DF8A478C8> could not be transformed and will be executed as-is. Please report this to the AutoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: 'NoneType' object has no attribute '_fields'
W1005 21:21:57.594096  8384 ag_logging.py:146] Entity <function Function._initialize_uninitialized_variables.<locals>.initialize_variables at 0x0000015DF8A478C8> could not be transformed and will be executed as-is. Please report this to the AutoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: 'NoneType' object has no attribute '_fields'
[       OK ] ForwardMethodsTest.testEntersNameScope_concreteFunction
[ RUN      ] ForwardMethodsTest.testFunctionType
[       OK ] ForwardMethodsTest.testFunctionType
[ RUN      ] ForwardMethodsTest.test_session
[  SKIPPED ] ForwardMethodsTest.test_session
[ RUN      ] ModuleTrackingTest.test_non_ctor_submodule
[       OK ] ModuleTrackingTest.test_non_ctor_submodule
[ RUN      ] ModuleTrackingTest.test_session
[  SKIPPED ] ModuleTrackingTest.test_session
[ RUN      ] ModuleTrackingTest.test_submodules
[       OK ] ModuleTrackingTest.test_submodules
[ RUN      ] NameScopeTest.test_memoized_in_tf2
[  SKIPPED ] NameScopeTest.test_memoized_in_tf2
[ RUN      ] NameScopeTest.test_not_memoized_in_tf1
[       OK ] NameScopeTest.test_not_memoized_in_tf1
[ RUN      ] NameScopeTest.test_session
[  SKIPPED ] NameScopeTest.test_session
[ RUN      ] TestModuleNaming.test_construct_in_scope
[       OK ] TestModuleNaming.test_construct_in_scope
[ RUN      ] TestModuleNaming.test_ctor_error_closes_name_scope
[       OK ] TestModuleNaming.test_ctor_error_closes_name_scope
[ RUN      ] TestModuleNaming.test_ctor_error_handles_ctor_not_opening_name_scope
[       OK ] TestModuleNaming.test_ctor_error_handles_ctor_not_opening_name_scope
[ RUN      ] TestModuleNaming.test_does_not_evaluate_property_methods
[       OK ] TestModuleNaming.test_does_not_evaluate_property_methods
[ RUN      ] TestModuleNaming.test_enters_name_scope_in_call
[       OK ] TestModuleNaming.test_enters_name_scope_in_call
[ RUN      ] TestModuleNaming.test_enters_name_scope_in_other_method
[       OK ] TestModuleNaming.test_enters_name_scope_in_other_method
[ RUN      ] TestModuleNaming.test_forward_method_closes_name_scope
[       OK ] TestModuleNaming.test_forward_method_closes_name_scope
[ RUN      ] TestModuleNaming.test_get_attr_doesnt_enter_name_scope
[       OK ] TestModuleNaming.test_get_attr_doesnt_enter_name_scope
[ RUN      ] TestModuleNaming.test_get_attribute_doesnt_enter_name_scope
[       OK ] TestModuleNaming.test_get_attribute_doesnt_enter_name_scope
[ RUN      ] TestModuleNaming.test_invalid_name
[       OK ] TestModuleNaming.test_invalid_name
[ RUN      ] TestModuleNaming.test_module_numbering_in_graph
2020-10-05 21:21:57.641573: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1618] Found device 0 with properties: 
name: GeForce RTX 2080 Ti major: 7 minor: 5 memoryClockRate(GHz): 1.65
pciBusID: 0000:73:00.0
2020-10-05 21:21:57.642107: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_100.dll
2020-10-05 21:21:57.642461: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cublas64_100.dll
2020-10-05 21:21:57.642817: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cufft64_100.dll
2020-10-05 21:21:57.643166: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library curand64_100.dll
2020-10-05 21:21:57.643517: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusolver64_100.dll
2020-10-05 21:21:57.643872: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusparse64_100.dll
2020-10-05 21:21:57.644258: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudnn64_7.dll
2020-10-05 21:21:57.644851: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1746] Adding visible gpu devices: 0
2020-10-05 21:21:57.645180: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1159] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-10-05 21:21:57.645553: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1165]      0 
2020-10-05 21:21:57.645841: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1178] 0:   N 
2020-10-05 21:21:57.646393: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1304] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 8686 MB memory) -> physical GPU (device: 0, name: GeForce RTX 2080 Ti, pci bus id: 0000:73:00.0, compute capability: 7.5)
[  SKIPPED ] TestModuleNaming.test_module_numbering_in_graph
[ RUN      ] TestModuleNaming.test_modules_not_numbered_in_eager
2020-10-05 21:21:57.659087: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1618] Found device 0 with properties: 
name: GeForce RTX 2080 Ti major: 7 minor: 5 memoryClockRate(GHz): 1.65
pciBusID: 0000:73:00.0
2020-10-05 21:21:57.659594: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_100.dll
2020-10-05 21:21:57.659938: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cublas64_100.dll
2020-10-05 21:21:57.660283: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cufft64_100.dll
2020-10-05 21:21:57.660621: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library curand64_100.dll
2020-10-05 21:21:57.660958: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusolver64_100.dll
2020-10-05 21:21:57.661300: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusparse64_100.dll
2020-10-05 21:21:57.661640: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudnn64_7.dll
2020-10-05 21:21:57.662186: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1746] Adding visible gpu devices: 0
2020-10-05 21:21:57.662484: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1159] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-10-05 21:21:57.662837: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1165]      0 
2020-10-05 21:21:57.663057: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1178] 0:   N 
2020-10-05 21:21:57.663531: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1304] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 8686 MB memory) -> physical GPU (device: 0, name: GeForce RTX 2080 Ti, pci bus id: 0000:73:00.0, compute capability: 7.5)
[       OK ] TestModuleNaming.test_modules_not_numbered_in_eager
[ RUN      ] TestModuleNaming.test_overridden_name_scope
[       OK ] TestModuleNaming.test_overridden_name_scope
[ RUN      ] TestModuleNaming.test_patched_callable
[       OK ] TestModuleNaming.test_patched_callable
[ RUN      ] TestModuleNaming.test_property
[       OK ] TestModuleNaming.test_property
[ RUN      ] TestModuleNaming.test_property_no_name_scope
[       OK ] TestModuleNaming.test_property_no_name_scope
[ RUN      ] TestModuleNaming.test_session
[  SKIPPED ] TestModuleNaming.test_session
[ RUN      ] TestModuleNaming.test_single_name
[       OK ] TestModuleNaming.test_single_name
[ RUN      ] TestModuleNaming.test_subclassed_module
[       OK ] TestModuleNaming.test_subclassed_module
[ RUN      ] TestModuleNaming.test_submodule_created_late
[       OK ] TestModuleNaming.test_submodule_created_late
[ RUN      ] VariableNamingTest.test_session
[  SKIPPED ] VariableNamingTest.test_session
[ RUN      ] VariableNamingTest.test_variable_names
[       OK ] VariableNamingTest.test_variable_names
[ RUN      ] VariableTrackingTest.test_session
[  SKIPPED ] VariableTrackingTest.test_session
[ RUN      ] VariableTrackingTest.test_supports_distributed_variables
[       OK ] VariableTrackingTest.test_supports_distributed_variables
[ RUN      ] VariableTrackingTest.test_trainable_variables
[       OK ] VariableTrackingTest.test_trainable_variables
[ RUN      ] VariableTrackingTest.test_trainable_variables_ignores_non_trainable
[       OK ] VariableTrackingTest.test_trainable_variables_ignores_non_trainable
[ RUN      ] VariableTrackingTest.test_variables
[       OK ] VariableTrackingTest.test_variables
----------------------------------------------------------------------
Ran 50 tests in 3.253s

OK (skipped=10)
